netbox/overlays/ocp-prod/config/extra.py

```diff
@@ -7,6 +7,6 @@
 REMOTE_AUTH_GROUP_SEPARATOR = ","
 REMOTE_AUTH_GROUP_SYNC_ENABLED = True
 REMOTE_AUTH_HEADER = "HTTP_X_FORWARDED_PREFERRED_USERNAME"
-REMOTE_AUTH_STAFF_GROUPS = ["/netbox-admin"]
-REMOTE_AUTH_SUPERUSER_GROUPS = ["/netbox-admin"]
+REMOTE_AUTH_STAFF_GROUPS = ["netbox-admins"]
```

Comment:
sorry I have been commenting after reviewing/merging. But I noticed that this is plural `-admins`. Is that change intentional?

Reply:
Yes, that was intentional -- we had two groups in keycloak, and I chose that one.